packages/client/src/templates/BaseSamenClient.ts

```diff
@@ -54,7 +54,17 @@ export class BaseSamenClient {
     const data = await result.json()
 
     if (!result.ok) {
-      throw errorParser(data)
+      if (result.status === 400) {
+        console.error(
+          `Result of RPC ${serviceName}.${functionName} has incorrect output`,
+          data,
+        )
```

Comment:
I'd say this is up to the user to log, right?

Reply:
Ah, nice catch, yeah.